### DISCUSSION TITLE
Redirect on ssg

### DISCUSSION THREAD

**Discussion by abbasudo** [statamic/cms#10206] [Q&A] [answered]
Hello there
I have added a redirect to my `web.php` file.
```php
Route::permanentRedirect('/blog', '/articles');
```
this works when I run `php artisan serve`. However, it doesn't work when generating SSG with `php please ssg:generate`. how to add a way to redirect SSG-generated sites? I'm using GitHub pages.

**Comment by jackmcdade** [comment, marked as answer]
Routes are not automatically generated. Here’s how you configure them for the SSG: https://github.com/statamic/ssg?tab=readme-ov-file#routes

**Reply by abbasudo** [reply]
You are the best
